### PR TITLE
 #14 added JavaDoc to com.jcabi.s3.Bucket.Prefixed

### PR DESCRIPTION
**Pull request by apocarteres**

### CODE REVIEW

**Comment by dmarkov** [comment]
Thanks, let me find someone who can review this pull request

**Comment by dmarkov** [comment]
@carlosmiranda review this one please

File: src/main/java/com/jcabi/s3/Bucket.java [hunk 1]
```diff
@@ -120,7 +120,10 @@ void remove(@NotNull(message = "S3 key can't be NULL") String key)
          */
         private final transient Bucket origin;
         /**
-         * Prefix.
+         * Prefix is a string which is used.
```
Comment (carlosmiranda) [inline src/main/java/com/jcabi/s3/Bucket.java]: I don't get what you're saying in this Javadoc description. Maybe it's better if you can provide an example.

**Comment by carlosmiranda** [comment]
@apocarteres One comment above.

File: src/main/java/com/jcabi/s3/Bucket.java (hunk 1 above)
Comment (apocarteres) [inline src/main/java/com/jcabi/s3/Bucket.java]: In JavaDoc mentioned the test, which explains <code>Prefixed</code> class logic well, doesn't it?
Take a look at <code>com.jcabi.s3.mock.MkBucketTest#listsOckets()</code>

Comment (carlosmiranda) [inline src/main/java/com/jcabi/s3/Bucket.java]: Someone who is reading this on a Javadoc page will not be able to see the source code directly. Better to show a short example right here.

Comment (apocarteres) [inline src/main/java/com/jcabi/s3/Bucket.java]: please take a look at new doc. I've added explanation and example

**Comment by carlosmiranda** [comment]
@rultor merge please

**Comment by yegor256** [comment]
@apocarteres did you try to render that javadoc and look at the HTML? it will look rather unreadable. try it yourself (`mvn clean site -Psite`). To make it more readable, you should use HTML markup inside.

**Comment by apocarteres** [comment]
@yegor256 no, i didn't. let me check

**Comment by apocarteres** [comment]
please take a look at human-readable version :)

**Comment by yegor256** [comment]
@apocarteres to be honest, the Java example looks good, while the text looks very difficult to read. Why so many words and the list? Can't you explain the same in a single paragraph of a plain text? It's very difficult to understand. It's an open source product, people will read this text. Let's try to make it look nice.

**Comment by apocarteres** [comment]
@yegor256 it should be nice now

**Comment by yegor256** [comment]
@apocarteres why do you put a normal text paragraph in `<pre>`? Look at this formatting: https://github.com/jcabi/jcabi-s3/blob/master/src/main/java/com/jcabi/s3/Ocket.java#L47-L62 Try to make yours similar :)

**Comment by carlosmiranda** [comment]
@apocarteres Any updates here?

**Comment by apocarteres** [comment]
@carlosmiranda looks like it done, but the original issue is not mine anymore

**Comment by carlosmiranda** [comment]
@apocarteres In that case, would it be all right if we close this pull request?

**Comment by apocarteres** [comment]
sure. thx for your time.

**Comment by dmarkov** [comment]
@carlosmiranda thanks, I just added **17 mins** to your account, payment `50954177`; +17 to your rating, your total score is [+2442](http://www.netbout.com/b/35023?open=rating)

**Pull request closed without merging**